sirepo/template/raydata.py

```diff
@@ -50,13 +50,7 @@ def stateless_compute_catalog_names(_):
 
 
 def stateless_compute_download_analysis_pdfs(data):
-    return template_common.JobCmdFile(
-        path=pkio.py_path(
-            _request_scan_monitor(
-                PKDict(method="download_analysis_pdfs", data=data)
-            ).path
-        )
-    )
+    return _request_scan_monitor(PKDict(method="download_analysis_pdfs", data=data))
```

Comment:
This code would need to add on the dataFileUri to the PKDict. That's a bit annoying, but again, this should be defined by the message that's expected. The use of data is so obscure at this point that it would be better to call it job_cmd_data or something.

Reply:
0f27df52a